.github/workflows/ci.yaml

```diff
@@ -0,0 +1,52 @@
+on:
+  pull_request:
+
+jobs:
+  dib-lint:
+    name: Run dib-lint
+    runs-on: ubuntu-latest
+
+    env:
+      ELEMENTS_DIR: elements
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Python 3.x
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+          cache: 'pip'
+
+      - name: Create & activate venv, install dib-lint
+        shell: bash
+        run: |
+          python -m venv .venv
+          source .venv/bin/activate
+          pip install --upgrade pip
+          pip install diskimage-builder
```

Comment:
we don't need it now, but it might be good to create a `test-requirements.txt` or pyproject.toml `dev` optional dependencies group 

it will make later integration with something like `tox` simpler, rather than specifying the same dependencies in both the github workflow and elsewhere

Reply:
good idea. done